### DISCUSSION TITLE
WARNING: This is a development server. Do not use it in a production deployment.

### DISCUSSION THREAD

**Discussion by NaAlSi2O6x** [Lost-MSth/Arcaea-server#47] [Q&A]
打开run.bat报错 WARNING: This is a development server. Do not use it in a production deployment.
  Use a production WSGI server instead.
怎么解决
![捕获](https://user-images.githubusercontent.com/103325956/162581322-e7c48a53-77b8-401f-b913-2156b49cd7ed.PNG)
？

**Comment by MBRjun** [comment]
尝试**以管理员身份运行**？
检查**端口是否被占用**？

**Reply by NaAlSi2O6x** [reply]
以管理员身份运行结果相同
端口没有被占用

**Reply by MBRjun** [reply]
> 以管理员身份运行结果相同 端口没有被占用

```
tasklist|findstr "80"
```

**Reply by NaAlSi2O6x** [reply]
谢谢可以启动了，但我怎么让游戏接这个服务器呢？

**Reply by NaAlSi2O6x** [reply]
不知道怎么看现在官服的地址是什么

**Reply by MBRjun** [reply]
> 不知道怎么看现在官服的地址是什么

简单抓个包就可以，拆安装包也行
现在是 arcapi-v2.lowiro.com

**Reply by NaAlSi2O6x** [reply]
如何抓包啊……我已用Charles在电脑和iPad上安装了证书 map remote setting也设置好了进入的还是官服
![捕获](https://user-images.githubusercontent.com/103325956/162602611-bff73cf1-abff-4eaf-b988-096f79663ddc.PNG)

**Reply by MBRjun** [reply]
> 如何抓包啊……我已用Charles在电脑和iPad上安装了证书 map remote setting也设置好了进入的还是官服 ![捕获](https://user-images.githubusercontent.com/103325956/162602611-bff73cf1-abff-4eaf-b988-096f79663ddc.PNG)

**https**

**Reply by NaAlSi2O6x** [reply]
无法连接至服务器

**Reply by MBRjun** [reply]
> 无法连接至服务器

SSL/TLS Pinning 干掉了吗

**Reply by NaAlSi2O6x** [reply]
可能没有 怎么干啊

**Reply by MBRjun** [reply]
> 可能没有 怎么干啊

https://www.google.com/search?q=iOS+SSL+Pinning+Bypass

**Reply by NaAlSi2O6x** [reply]
谢谢。能力范围外了
